composer.json

```diff
@@ -13,8 +13,8 @@
         }
     ],
     "require": {
-        "php": ">=5.3.1",
-        "symfony/process": "~2.1|~3.0"
+        "php": ">=7.1",
```

Comment:
This PR is dropping support for old versions, not adding support for new versions of PHP (which were already allowed by the range